pkg/minikube/node/cache.go

```diff
@@ -102,11 +101,12 @@ func doCacheBinaries(k8sVersion string) error {
 // BeginDownloadKicArtifacts downloads the kic image + preload tarball, returns true if preload is available
 func beginDownloadKicArtifacts(g *errgroup.Group) {
 	glog.Info("Beginning downloading kic artifacts")
-	if !image.ExistsImageInDaemon(kic.BaseImage) {
+	baseImage := viper.GetString("base-image")
```

Comment:
just an optional nit: this string is in danger of if we change the string, it will be left unseen. but optional.

Reply:
sure, i'll address that in my wip PR